deftree/build_deftree.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/TuneLab/go-truss/deftree/svcparse"
-	"github.com/TuneLab/go-truss/truss/protostage"
+	"github.com/TuneLab/go-truss/truss/protostuff"
```

Comment:
Nah, I didn't look that closely. Avoid intentional name collisions. :) Just something that describes what it does better than "stuff."